ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -380,6 +380,16 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-upi-proxy
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-user-provisioned-dns
```

Comment:
```suggestion
  as: e2e-aws-ovn-user-provisioned-dns
```

Reply:
Oh good call

Reply:
This needs to be fixed in the GCP job too. I can take care of that.